.github/workflows/wheels.yml

```diff
@@ -57,18 +68,22 @@ jobs:
           # of RAM and OOMs while building if we give it the default number of
           # workers (2 * NUM_CPUs).
           export CIBW_ENVIRONMENT="MAX_JOBS=4 \
-                  TRITON_WHEEL_NAME=triton-nightly \
```

Comment:
We should keep wheel name triton rather then triton-nightly.
Nightly version will have git hash appended to it. Release version will only have version number,